### ISSUE TITLE
NTR: MCAD enzyme low in fibroblasts or other cells (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9069]
New term request MCAD enzyme low in fibroblasts or other cells (segal_020930154633): Medium chain acyl CoA dehydrogenase

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000356
- label: Reduced tissue medium-chain acyl-CoA dehydrogenase activity
- parents: [Abnormal tissue enzyme activity (HP:6000231)](https://hpo.jax.org/app/browse/term/HP:6000231)
- pmid: PMID:2393404
- def: Concentration or activity of medium-chain acyl-CoA dehydrogenase (EC 1.3.8.7;MCAD) in tissues is below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9069
- synonyms: Reduced MCAD activity in cultured fibroblasts(NONE)|Reduced medium-chain acyl-CoA dehydrogenase activity in cultured fibroblasts(NONE)
